waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -1822,7 +1839,8 @@
 (deftest test-pod->ServiceInstance
   (let [api-server-url "https://k8s-api.example/"
         base-scheduler {:api-server-url api-server-url
-                        :container-running-grace-secs 120}
+                        :container-running-grace-secs 120
+                        :restart-expiry-threshold 10}
```

Comment:
I added `:restart-expiry-threshold 10` to the base-scheduler here because I found it confusing in the tests below how every one had this value explicitly set, even if the test doesn't really care about the expiry threshold. I think treating 10 as the "default" makes the intent more clear, especially with the sidecar-related tests.